test/common/tcp_proxy/tcp_proxy_test.cc

```diff
@@ -1564,9 +1564,15 @@ TEST_F(TcpProxyTest, DEPRECATED_FEATURE_TEST(AccessLogBytesRxTxDuration)) {
   upstream_callbacks_->onEvent(Network::ConnectionEvent::RemoteClose);
   filter_.reset();
 
+#if !defined(WIN32)
   EXPECT_THAT(access_log_data_,
               MatchesRegex(
                   "bytesreceived=1 bytessent=2 datetime=[0-9-]+T[0-9:.]+Z nonzeronum=[1-9][0-9]*"));
```

Comment:
If we don't care to preserve the `nonzeronum=[1-9][0-9]*` expression, we are happy to collapse this logic above and use the `\\d+` expressions below

Reply:
Nevermind, \d is unsupported on posix regex.

Reply:
Maybe we could preprocess the regex in Windows to catch these cases?

Reply:
A single "simple re" regex can't really express this without backrefs, character classes etc. It would be a second test. The solution is to stop using simple re, but that requires the Google Test project to support Google RE2, or requires us to include PCRE across the board. The patch we've pushed quietly gets the simple re tests out of the way on Windows if PCRE is supported later.